package.json

```diff
@@ -182,10 +180,10 @@
     "ncp": "^2.0.0",
     "ng-packagr": "4.7.1",
     "node-sass": "4.13.0",
-    "protractor": "^5.4.2",
-    "protractor-browser-logs": "^1.0.423",
-    "protractor-html-reporter-2": "^1.0.4",
-    "protractor-retry": "^1.2.0",
+    "protractor": "6.0.0-beta",
```

Comment:
protractor 6.0 is not stable yet.
following their gitter channel and github issues, I saw that they don't recommend upgrading to 6.0 yet.